pkgs/os-specific/linux/sssd/default.nix

```diff
@@ -0,0 +1,65 @@
+{ stdenv, fetchurl, pkgs, augeas, bind, c-ares, cyrus_sasl, ding-libs, libnl,
+  libunistring, nss, samba, libnfsidmap, doxygen, python, python3,
+  pam, popt, talloc, tdb, tevent, pkgconfig, ldb, openldap, pcre, kerberos,
+  cifs_utils, glib, keyutils, dbus, fakeroot, libxslt, libxml2,
+  docbook_xml_xslt, ldap, systemd, nspr, check, cmocka,
+  uid_wrapper, nss_wrapper, docbook_xml_dtd_44, ncurses, Po4a }:
+
+let
+  name = "sssd";
+  version = "1.13.3";
+in stdenv.mkDerivation {
+  inherit name;
+  inherit version;
+
+  src = fetchurl {
+    url = "https://fedorahosted.org/released/${name}/${name}-${version}.tar.gz";
+    sha256 = "3fd8fe8e6ee9f50b33eecd1bcccfaa44791f30d4e5f3113ba91457ba5f411f85";
+  };
+
+  preConfigure = ''
+    cd "$NIX_BUILD_TOP/$name-$version"
```

Comment:
To be honest, I didn't even look at @benwbooth's original submission properly. I'm using this opportunity to clean up all of it. Do you think it's a good idea to update to the latest SSSD before finalizing the submission? There seem to have been a couple of releases ever since.